pycalphad/core/constants.py

```diff
@@ -6,7 +6,7 @@
 # Force zero values to this amount, for numerical stability
 MIN_SITE_FRACTION = 1e-12
 # For each phase pair with composition difference below tolerance, eliminate phase with largest index
```

Comment:
I think I was unclear. The comment in line 8 should be updated because we don't alway use COMP_DIFFERENCE_TOL to remove phases, we use it to add phases as well.